<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>System Tray</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="chkTray">
            <property name="text">
             <string>Show system &amp;tray icon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkStartHidden">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Start &amp;with hidden main window</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Initial Status</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="cmbInitialStatus">
            <item>
             <property name="text">
              <string>Offline</string>
             </property>
             <property name="icon">
              <iconset resource="../main.qrc">
               <normaloff>:/gui/icons/im-user-offline.png</normaloff>:/gui/icons/im-user-offline.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Online</string>
             </property>
             <property name="icon">
              <iconset resource="../main.qrc">
               <normaloff>:/gui/icons/im-user.png</normaloff>:/gui/icons/im-user.png</iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>170</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Server</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Hostname:</string>
              </property>
              <property name="buddy">
               <cstring>txtHostname</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtHostname"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="buddy">
               <cstring>spbPort</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spbPort">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65536</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Authentication</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Username:</string>
              </property>
              <property name="buddy">
               <cstring>txtUsername</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtUsername"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="chkSavePassword">
              <property name="text">
               <string>Save Password</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Password:</string>
              </property>
              <property name="buddy">
               <cstring>txtPassword</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtPassword">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>127</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Events</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Notifications</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="chkAnimateTray">
            <property name="text">
             <string>Animate system tray icon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkPopups">
            <property name="text">
             <string>Show popups</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>244</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkStartHidden</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSavePassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSavePassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
